src/bus/webSocketConnection.spec.js

```diff
@@ -74,11 +76,13 @@ describe('Bus/WebSocketConnection', function() {
             id: jsonRpcId
           });
 
-          promise = ws.authorize(token);
-
           webSocketServer.emit(
             'message',
-            JSON.stringify({ jsonrpc: '2.0', id: jsonRpcId, result: null })
+            JSON.stringify({
```

Comment:
To test this unit, we don't really need to do this anymore -- can test it by invoking the method directly. The mechanism for how onmessage works isn't even in this method anymore

Reply:
This was marked resolved but there's no comment about it and nothing has changed?

Reply:
There was no comment before, but you have the same situation in the publish -- it's no longer testing just a singular unit since it depends on `onMessage` to do the handling. Should be testing that that message handler is set up correctly and what the message handler itself does

Reply:
I think I just misunderstood what you meant the first time. Should be updated now